vectortile/src/main/scala/geotrellis/vectortile/package.scala

```diff
@@ -99,4 +99,11 @@ package geotrellis
   * @author cwoodbury@azavea.com
   * @version 2.1
   */
-package object vectortile {}
+
+import geotrellis.vector.Geometry
+
+package object vectortile {
+  implicit class withGeometryMethods[G <: Geometry](val self: G) extends AnyVal {
+    private[vectortile] def normalize(enabled: Boolean = true): G = { if(enabled) { self.jtsGeom.normalize() }; self }
```

Comment:
Ew. But this seems fine given the API provided to normalize.

Reply:
Lol.  But if it works.... The vectortile package is already a bit gross.

Reply:
Im going to remove this mess